Formula/root.rb

```diff
@@ -22,6 +22,7 @@ class Root < Formula
   depends_on "ninja" => :build
   depends_on "cfitsio"
   depends_on "davix"
+  depends_on "expat" if MacOS.version < :big_sur
```

Comment:
```suggestion
  uses_from_macos "expat", since: :big_sur
```

Reply:
Sounds good, though assuming this will still work as is to see if it just passes? I might be able to revert 
b0213f6 if this does pass, so I'll be updating it at least once. I would have made it draft if it was my PR.